eras/alonzo/formal-spec/mir.tex

```diff
@@ -0,0 +1,415 @@
+\section{MIR Certificates}
+\label{sec:mir}
+
+\newcommand{\InstantaneousRewards}{\type{InstantaneousRewards}}
+\newcommand{\MIRPot}{\type{MIRPot}}
+\newcommand{\ReservesMIR}{\type{ReservesMIR}}
+\newcommand{\TreasuryMIR}{\type{TreasuryMIR}}
+
+In the Alonzo era, MIR certs have:
+
+\begin{itemize}
+  \item Slightly different behavior -
+  prior to the Alonzo era, MIR certificates within the same epoch override
+  values for repeated stake addresses.
+  In the Alonzo era, repetitions are handled by adding the values.
+  Moreover, negative values are allowed, so long as the change never results in
+  an overall negative sum.
+  See Figure \ref{fig:dcert-mir}.
+  \item Extended functionality -
+  MIR certificates can now be used to transfer funds between the reserves and the treasury.
+  See Figure \ref{fig:dcert-mir-trans}.
+\end{itemize}
+
+Figure \ref{fig:defs:alonzo-mir} describes the necessary changes to the types.
+Note that $\fun{mirTarget}$ was named $\fun{credCoinMap}$ in the Shelley specification.
+Figure \ref{fig:dcert-mir} is a modification to the DELEG rule, and
+Figure \ref{fig:dcert-mir-trans} is an addition to the DELEG rule.
+Lastly, Figure \ref{fig:rules:mir} is the updated MIR rule.
+
+
+\begin{figure*}[htb]
+  \emph{Derived types}
+  %
+  \begin{equation*}
+    \begin{array}{llr}
+      \InstantaneousRewards =
```

Comment:
Spacing for the `=` sign is wrong, and I think for `\times` as well

Reply:
yea, it's pretty ugly. I just tried using the array settings used in `TxWitness`, which does look better, but still a bit off.

Reply:
I think it gets the seal of 'good enough' :)

Reply:
:sweat_smile: